tests/common/iot_tests_network.c

```diff
@@ -107,12 +108,18 @@ static bool bleEnabled = false;
 
         if( bInitBLE == false )
         {
-            xStatus = IotBle_Init();
-
-            if( xStatus == eBTStatusSuccess )
-            {
-                bInitBLE = true;
-            }
+        	/* Initialize low level drivers for BLE */
+        	xStatus = bleStackInit( );
```

Comment:
Does it also make sense to teardown ble stack after all ble stacks ? That will be a good practice as we dont mandate anywhere multiple testgroups cannot be run together.

Reply:
Yes but that would be a separate commit. Right now we don't do it.